modules/virt-creating-a-vm-from-a-default-os-image.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-//images.adoc
+
+[id="virt-creating-a-vm-from-a-default-os-image_{context}"]
+= Creating a virtual machine from a default OS image
+
+After you have created a Persistent Volume Claim (PVC) for default OS images, you create a new virtual machine or virtual machine template from a default OS image that you uploaded to the PVC.
+
+[NOTE]
+====
+You can create virtual machines and virtual machine templates from default OS images with non-administrator user privileges. You need administrator privileges to upload and save default OS images to PVCs.
+====
+
+.Prerequisites
+
+* A PVC must exist that supports default OS images.
+* When you created the PVC, you selected the *Attach this disk to a VirtualMachine operating system* check box.
+
+.Procedure
+
+. In the {product-title} Web Console, click *Workloads > Virtualization* in the sidebar menu. The *Virtualization* page displays. Use onscreen Help or existing documentation to create virtual machine templates.
```

Comment:
There's 4 instances of "Web Console" init capped like this, but typically this is just lowercase "web console" elsewhere in the repo.

Reply:
Not sure about usage of "onscreen". Not seeing other usage in the repo for this spelling or "on-screen"... Maybe just "web console **Help**" (if the UI item is indeed labeled "Help")?

Reply:
Thanks. Fixed. Good to know re: web console Help. Wasn't sure if Help was capped like that anymore (when there was an actual Help menu it obviously was :-))